src/components/PoiTitle.jsx

```diff
@@ -37,7 +38,9 @@ const PoiTitle = ({ poi, withAlternativeName }) => {
 
   // Location / address
   return <div className="poiTitle">
-    <h2 className="poiTitle-main u-text--heading6">{title || subclass}</h2>
+    <h2 className={classnames('poiTitle-main u-text--heading6', { 'u-ellipsis': inList })}>
```

Comment:
great, but shouldn't we apply the u-ellipsis class without inList condition ?

Reply:
Nope, as we want the full name of the POI to be available when displaying the details of a single POI.
The ellipsis is here on lists so no POI takes too much vertical place compared to the others, for regularity and visual balance.